types/index.d.ts

```diff
@@ -71,6 +72,11 @@ type Props = {
    */
   renderers?: Partial<Renderers>
 
+  /**
+   * Parameters for the function [marked.use](https://marked.js.org/using_pro#use)
+   */
+  use?: Parameters<typeof marked.use>
```

Comment:
Since this can add renderers, if you're using TypeScript it would conflict a bit with the `Renderers` type above. Maybe extending said type with something like `& Record<string, InstantiableSvelteComponentTyped>` would work?

Reply:
Since we don't use the renderer either way, maybe we should discourage passing one with the use parameter:
```typescript 
use?: Omit<marked.MarkedExtension, 'renderer'>[]
```

Reply:
I just realized you were not talking about the use-type but the the Renderers type (just as you've written…). And there you are right, of course. The type would have to be added.